projects/2019_12_14_pkt-whitepaper-website.md

```diff
@@ -372,3 +372,589 @@ The applicant understands that the Network Steward is not a legal entity and no
+./configure
+make
+npm install
+ADD CODE
+
+Once you’ve completed building PacketCrypt, you can begin mining. Learn More
+
+
+##Begin Mining
+There are two types of mining available and pools should typically make about 50% of their payment to their announcement miners and 50% of their payment to their block miners.
+
+Devices with low available memory and/or slow internet connection should prefer to do announcement mining while devices with high speed internet and lots of available memory may prefer block mining.
+
+These examples have you mining in the Gridfinity mining pool and also giving your coins to cjd, make sure you update the --paymentAddr appropriately unless you wish to make a donation.
+
+##Announcement mining:
+
+node ./annmine.js --threads 2 --paymentAddr=pDSxcZunaUSUSxHrL6r8zpGJvoEropJ3Es http://pool.gridfinity.com/master
+
+##Block mining:
+
+node ./blkmine.js --threads 2 --paymentAddr=pDSxcZunaUSUSxHrL6r8zpGJvoEropJ3Es http://pool.gridfinity.com/master
+
+##Resources
+Telegram: https://t.me/pktproject
+Matrix: #pkt:matrix.org
+IRC: #pkt@freenode.net
+Suggest an edit to this page
+Main
+
+
+
+{PAGE}
+#Network Steward 
+By consensus rule, 20% of every newly mined block goes to an elected address, which is the Network Steward. The Network Steward is currently a 5 person group that is dedicated to helping build the PKT ecosystem through competitive grants for open source projects.
+Network Steward Charter
+
+The objective of The Network Steward is to reduce barriers of entry to support the PKT Network in the pursuit of increased individual autonomy and privacy.
+
+Key aspects of this objective include:
+
+Financing the development of open source networking software
+Lobbying for improved regulatory environment for small network operators and more generally, for internet freedom, privacy, and decentralization
+Purchasing property such as proprietary software or radio frequency spectrum rights in order to put these things in the commons
+
+In order to achieve these goals, The Network Steward will occasionally meet to review funding proposals and choose which projects to grant PKT to.
+How to Make a Funding Proposal
+
+Getting a proposal accepted is much like getting a pull request merged in a project. You start by socializing what you want to do with the community (try the IRC or Matrix chats) and when you have rough consensus, you move that conversation to a formal proposal.
+
+To make a proposal, you must create a pull request to the ns-projects repository which adds a new project markdown file as per the project template and name it according to the naming convention defined in the repository readme.
+
+Periodically The Network Steward will hold a meeting and decide on which proposals should be funded. The minutes of these meetings are published in the the ns-projects repository along with the projects and their status.
+
+Button:
+Submit a Proposal
```

Comment:
I  would like to see a link to the proposal template (https://github.com/pkt-cash/ns-projects/blob/master/projects/template.md)